homework.rb

```diff
@@ -1,6 +1,64 @@
-VARIANT = 0
+VARIANT = 5
+
+LETTER_START_POINT = 65
+LETTER_END_POINT = 90
+# pattern = 'AC-1234567-ABC'
+
+def sequence_nature(pattern='AC-1234567-ABC', index=0)
```

Comment:
if u are using a single element of pattern here, is there need to pass whole pattern with needed index?